pkg/controller/cronjob/utils.go

```diff
@@ -198,6 +200,8 @@ func getLatestMissedScheduleBinarySearch(startWindow time.Time, endWindow time.T
 func getJobFromTemplate(sj *batchv1beta1.CronJob, scheduledTime time.Time) (*batchv1.Job, error) {
 	labels := copyLabels(&sj.Spec.JobTemplate)
 	annotations := copyAnnotations(&sj.Spec.JobTemplate)
+	annotations[jobScheduledTimeAnnotationKey] = metav1.NewTime(scheduledTime).String()
```

Comment:
The internal Time type appears to use https://golang.org/pkg/time/#Time.String

Do you definitely want to do timestamp parsing on the consumer side of this annotation?

Reply:
Good call. Going to need to do time stamp deserialization / parsing on the consumer side of this annotation. Changed to be explicitly RFC1123Z format